code/modules/projectiles/ammunition/ammo_kits.dm

```diff
@@ -50,85 +50,92 @@
-			"Scrap 9mm speedloader (6 ammo, 3 points)" = list(3, /obj/item/ammo_magazine/speed_loader_pistol_35/scrap),
-			"Scrap 9mm ammo box (30 ammo, 12 points)" = list(12, /obj/item/ammo_magazine/ammobox/pistol_35/scrap),
-			"9mm ammo pile (15 ammo, 8 points)" = list(8, /obj/item/ammo_casing/pistol_35/prespawned),
-			"9mm standard magazine (empty, 10 ammo, 5 points)" = list(5, /obj/item/ammo_magazine/pistol_35/scrap/empty),
-			"9mm extended magazine (empty, 16 ammo, 10 points)" = list(10, /obj/item/ammo_magazine/highcap_pistol_35/empty),
-			"9mm SMG magazine (empty, 32 ammo, 12 points)" = list(12, /obj/item/ammo_magazine/smg_35/empty),
-			"9mm ammo box (30 ammo, 26 points)" = list(26, /obj/item/ammo_magazine/ammobox/pistol_35)
+			"5 points, 15 ammo: Scrap 9mm ammo pile" = list(5, /obj/item/ammo_casing/pistol_35/scrap/prespawned),
+			"3 points, 6 ammo: Scrap 9mm speedloader" = list(3, /obj/item/ammo_magazine/speed_loader_pistol_35/scrap),
+			"12 points, 30 ammo: Scrap 9mm ammo box" = list(12, /obj/item/ammo_magazine/ammobox/pistol_35/scrap),
+			"8 points, 15 ammo: 9mm ammo pile" = list(8, /obj/item/ammo_casing/pistol_35/prespawned),
+			"18 points, 30 ammo: 9mm ammo box" = list(18, /obj/item/ammo_magazine/ammobox/pistol_35),
+			"5 points: EMPTY 9mm pistol magazine" = list(5, /obj/item/ammo_magazine/pistol_35/empty),
+			"7 points: EMPTY 9mm extended magazine" = list(7, /obj/item/ammo_magazine/highcap_pistol_35/empty),
+			"9 points: EMPTY 9mm SMG magazine" = list(12, /obj/item/ammo_magazine/smg_35/empty)
 			),
 		CAL_MAGNUM = list(
-			"Scrap 10mm magnum ammo pile (6 ammo, 3 points)" = list(3, /obj/item/ammo_casing/magnum_40/scrap/prespawned),
-			"Scrap 10mm magnum ammo box (30 ammo, 15 points)" = list(15, /obj/item/ammo_magazine/ammobox/magnum_40/scrap),
-			"Scrap 10mm magnum speedloader (6 ammo, 5 points)" = list(4, /obj/item/ammo_magazine/speed_loader_magnum_40/scrap),
-			"10mm magnum ammo pile (6 ammo, 7 points)" = list(7, /obj/item/ammo_casing/magnum_40/prespawned),
-			"10mm magnum standard magazine (empty, 10 ammo, 5 points)" = list(3, /obj/item/ammo_magazine/magnum_40/empty),
-			"10mm magnum ammo box (30 ammo, 26 points)" = list(26, /obj/item/ammo_magazine/ammobox/magnum_40)
+			"3 points, 6 ammo: Scrap 10mm ammo pile" = list(3, /obj/item/ammo_casing/magnum_40/scrap/prespawned),
+			"15 points, 30 ammo: Scrap 10mm ammo box" = list(15, /obj/item/ammo_magazine/ammobox/magnum_40/scrap),
+			"5 points, 6 ammo: Scrap 10mm speedloader" = list(5, /obj/item/ammo_magazine/speed_loader_magnum_40/scrap),
+			"7 points, 6 ammo: 10mm magnum ammo pile" = list(7, /obj/item/ammo_casing/magnum_40/prespawned),
+			"26 points, 30 ammo: 10mm ammo box" = list(26, /obj/item/ammo_magazine/ammobox/magnum_40),
+			"7 points: EMPTY 10mm pistol magazine" = list(7, /obj/item/ammo_magazine/magnum_40/empty),
+			"10 points: EMPTY 10mm SMG magazine" = list(10, /obj/item/ammo_magazine/smg_magnum_40/empty)
+			),
+		CAL_50 = list(
```

Comment:
against 12mm added

Reply:
12mm is literally the only caliber that the ammo kit cannot craft, why should it be excluded? Find a 12mm gun, no guild, no spec, get fucked and never use it because no mag/ammo???
As a reminder you can still craft .70 GYROJET AMMO, BUT NOT 12mm, tell me which is more common loot, a 12mm gun or the goddamn Emperor Gyrojet pistol?